### PR TITLE
Update community page with latest webinar and community meeting video

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
I worked on Updating the community page with the latest webinar and community meeting video.

image

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -369,15 +370,13 @@ <h2 class="section-header fw-600 mb-4">
...
+                            src="https://www.youtube.com/embed/8gbj9c_wb9Y"
                             title="YouTube video player"
                             frameborder="0"
                             allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                             allowfullscreen
                           ></iframe>
-                          <p class="fw-600 mb-0 mt-2">
-                            OpenMetadata 0.12.0 Release
-                          </p>
+                          <p class="fw-600 mb-0 mt-2">October 27th, 2022</p>
```
Comment (chirag-madlani) [inline community.html:379]: Not sure but can we use an array and loop over it to reduce change at multiple places? 
And others can update the data easily I feel

Comment (Sachin-chaurasiya) [reply community.html:379]: Sure @chirag-madlani, We can improve the code in another patch.

**Review by chirag-madlani — APPROVED** [review]

**Review by ShilpaVernekar — APPROVED** [review]
@Sachin-chaurasiya - Thanks for the changes. Please merge these changes for now.
I'll work with Sanket B and get the Community Meeting YouTube thumbnails edited, so the text does not get cut out. Also, I'll work on the updated content for the OM Community page.

Also @harshach - In the OM Community page, we have 2 sections for videos: 1) Webinars and 2) Community Meetings...... Thinking of changing that to 3 sections, with the 3rd one being Release Videos (0.12, 0.11, etc).... Ideally those are the post-release community meeting videos. Also, thinking of creating YouTube playlists for Webinar, Releases, Community Meetings, Feature Videos...... Then use those playlist URLs to link at the end of these sections on OM Community page for View more....

**Comment by ShilpaVernekar** [comment]
Informed @Ashish8689 about changing the text for the Monthly Meeting and Webinar section.
https://docs.google.com/document/d/1O7IawHaW79vxTERouE71-Li7B-UTb7vTwi0nz9O_Zo4/edit#

**Pull request merged**